microscopium/preprocess.py

```diff
@@ -171,7 +173,7 @@ def maxes(fns):
     return maxes
 
 
-def stretchlim(im, bottom=0.01, top=None, mask=None):
+def stretchlim(im, bottom=0.01, top=None, mask=None, in_place=False):
```

Comment:
good idea, especially with the new reservoir sampling (fewer chance of fluff ruining the whole normalisation).